moveit_ros/occupancy_map_monitor/src/occupancy_map_server.cpp

```diff
@@ -35,27 +35,31 @@
 /* Author: Jon Binney, Ioan Sucan */
 
 #include <memory>
-#include <ros/ros.h>
+#include <rclcpp/rclcpp.hpp>
 #include <tf2_ros/transform_listener.h>
 #include <moveit/occupancy_map_monitor/occupancy_map_monitor.h>
 #include <octomap_msgs/conversions.h>
+#include <boost/bind.hpp>
 
-static void publishOctomap(ros::Publisher* octree_binary_pub, occupancy_map_monitor::OccupancyMapMonitor* server)
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros.occupancy_map_server");
```

Comment:
```suggestion
static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_ros_occupancy_map_server");
```

Nit: @henningkayser Which one should we use .?

Reply:
Ok, now that we have multiple files in this package I would strongly prefer `lib_name.file_name` for all files. It's better to be more explicit about this and makes scoping logger levels easier